src/get.token.ropc.ts

```diff
@@ -0,0 +1,35 @@
+import { AuthenticationClient } from './authentication/authentication.client';
+import { IToken } from './contracts/token';
+import * as dotenv from 'dotenv';
+
+const main = async () => {
+
+  dotenv.config();
+
```

Comment:
Break the process if any configuration parameter is missing. In the early version of the server we used `envalid` package to validate env config.